lib/algorithms/rsa-util.js

```diff
@@ -49,7 +50,25 @@ function convertToJWK(key, isPublic) {
   return result;
 }
 
+var pemCache = new WeakMap();
```

Comment:
This code base still runs on platforms where WeakMap is not available.

Personally, I see caching as a nice-to-have, so maybe forgo it for now in favor of a method of caching it in the key representation ...